doc/source/user_guide/launching_ansys_fluent.rst

```diff
@@ -193,4 +193,27 @@ For distributed parallel processing, you usually pass both parameters:
       version="3d",
       mode="solver",
       additional_arguments="-t16 -cnf=m1:8,m2:8",
-   )
\ No newline at end of file
+   )
+
+Using the PYFLUENT_ADDITIONAL_ARGUMENTS environment variable
```

Comment:
I don't think you need this doc here at all unless you want to go through the library and document all the other environment variables that are used for various purposes.

Generally we've not really been documenting them by design, because we don't want to encourage library users to be using them.  i.e. they are kind of meant to be hidden.

Reply:
In some cases they have a specific uses such as:

- point the library at a local development version of Fluent
- control some specific detail of the CI/CD pipeline that is part of building and testing the library, but not using it

Neither of these things is needed to _use_ the library right. Fundamentally I don't think there should be any behaviour of the _customer facing_ API calls that depend on any environment variable setting.  Environment variables should not be part of an API IMHO.